$NetBSD: patch-auto_make,v 1.1 2023/05/17 16:44:51 osa Exp $

Use pipe as a delimiter for sed(1) to fix build for libnjs target.

sed(1) command line utility may fail with the following error:
sed: 1: "s, at EXTRA_LIBS@,-lm   -L ...": bad in substitute command: '-'
when a replacement for @EXTRA_LIBS@ contains a comma symbol.

--- auto/make.orig	2023-05-11 23:24:57 UTC
+++ auto/make
@@ -320,11 +320,11 @@ cat << END >> $NJS_MAKEFILE
 pc: $NJS_BUILD_DIR/njs.pc
 
 $NJS_BUILD_DIR/njs.pc: $NJS_BUILD_DIR/njs_auto_config.h
-	sed -e "s,@PREFIX@,$(pwd)/$NJS_BUILD_DIR," \\
-		-e "s,@LIBDIR@,$(pwd)/$NJS_BUILD_DIR," \\
-		-e "s,@CFLAGS@,-I$(pwd)/$NJS_BUILD_DIR -I$(pwd)/src," \\
-		-e "s,@VERSION@,\$(NJS_VER)," \\
-		-e "s,@EXTRA_LIBS@,-lm $NJS_LIBS $NJS_LIB_AUX_LIBS," \\
+	sed -e "s|@PREFIX@|$(pwd)/$NJS_BUILD_DIR|" \\
+		-e "s|@LIBDIR@|$(pwd)/$NJS_BUILD_DIR|" \\
+		-e "s|@CFLAGS@|-I$(pwd)/$NJS_BUILD_DIR -I$(pwd)/src|" \\
+		-e "s|@VERSION@|\$(NJS_VER)|" \\
+		-e "s|@EXTRA_LIBS@|-lm $NJS_LIBS $NJS_LIB_AUX_LIBS|" \\
 		src/njs.pc.in > \$@
 END
 
